creator-cards/barret-blake.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Barret Blake",
+  "sectionTitle": null,
+  "countryCode": "US",
+  "slogan": "",
+  "type": "CREATOR",
+  "bio": "I wear many hats. Father, husband, Microsoft MVP, developer, gamer, blogger, speaker, model railroader, and so forth. I’ve been blessed greatly by this industry, and so this site is one way I try to give a little bit back. When I can find the time, I add a bit of my coding knowledge and experience to share with whoever might find it of worth.",
+  "tags": ".NET, C#, Azure, Blazor, DevOps, Power Automate, Serverless, Cloud, MSSQL, SQL, .NET Core, Microsoft, Azure Data Factory, ASP.NET",
+  "socials": {
+    "youtube": "https://www.youtube.com/@barretcodes",
+    "linkedin": "https://www.linkedin.com/in/barretblake",
+    "twitter": "https://twitter.com/barretblake",
+    "github": "",
+    "mastodon": "https://mstdn.social/@barret",
+    "sessionize": "https://sessionize.com/barretblake/"
+  },
+  "channels": [
+    {
+      "name": "Blog",
+      "url": "https://barretblake.dev"
+    }
+  ],
+  "feeds": [{
+      "type": "LinkTagRssType",
+      "url": "https://barretblake.dev/tag/dotnet/feed/"  
+    },{
+      "type": "LinkTagRssType",
```

Comment:
Please remove the unrelated feeds, we might have a place to add them at a later stage. Happy to merge that afterwards